free-1.md

```diff
@@ -57,12 +153,26 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Probably upwards of 70% of their time on the systems aspect.  While many of 
+the reaction mechanisms are straightforward, data structure design would be
+very non-trivial, as would be transcribing the mechanisms to algorithms.  The 
+language would probably be relatively trivial.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Very big and ambitious.  It takes a ton of domain knowledge and programming
+expertise.
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This is probably not a great idea for a project.  It is very large-scale, and 
+really hard.  Furthermore, most of the time working on it would not be focused
+on the language design.  That being said, the complexity of it all could lend 
+itself to a very interesting project, and the language design decisions to make
+everything work flexibly but also be easily extensible would be very interesting
+as well.
```

Comment:
I think this could be a really cool idea and project. Is there some small piece that you think you could tackle this semester? Do you have enough domain expertise to implement the semantics? If so, I say go for it!